### PR TITLE
Updates partials

### PR DESCRIPTION
**Pull request by aprilrieger**
Ref ticket: https://github.com/scientist-softserv/oral-history/issues/19

Changes to the files from 7.10 to 7.33.1: https://github.com/projectblacklight/blacklight/compare/v7.10.0...v7.33.1

Checkout main branch
get `.env.development` here: https://start.1password.com/open/i?a=LTLZ652TT5H5FHMYMASSH7PIXM&v=huuakin4bu4xanlhktv42qheam&i=ok6rwracafccnhrhur7usyfwoi&h=scientist.1password.com

```
gem install stack_car -v0.16.0
gem install dory

dory up
sc build
sc up
```

navigate to http://oralhistory.test/users/sign_in && login: creds in the .env

navigate to http://oralhistory.test/admin

run importer OR run in web container `rake db:migrate import[10]` [#] <— amount of works you import

Need troubleshooting tips?
If you do a `dc down -v`, odds are you will want to do a `rm -rf tmp/*`

List of views that have been upgraded from Blacklight v7.10 - 7.33.1 that our application currently has in the applications local files.

https://github.com/projectblacklight/blacklight/compare/v7.10.0...v7.33.1

- [x] app/views/catalog/_constraints.html.erb
- [x] app/views/catalog/_email_form.html.erb
- [x] app/views/catalog/_facet_layout.html.erb
- [x] app/views/catalog/_facets.html.erb
- [x] app/views/catalog/_home_text.html.erb
- [x] app/views/catalog/_index.html.erb
- [x] app/views/catalog/_search_form.html.erb
- [x] app/views/catalog/_search_results.html.erb
- [x] app/views/catalog/_show_tools.html.erb
- [x] app/views/catalog/_start_over.html.erb
- [x] app/views/catalog/show.html.erb
- [x] app/views/search_history/index.html.erb
- [x] app/views/shared/_header_navbar.html.erb
- [x] app/models/search_builder.rb
- [x] app/controllers/concerns/blacklight/bookmarks.rb
- [x] app/services/blacklight/field_retriever.rb
- [x] All full_text blacklight partials
- [x] All interviewee blacklight partials

Steps:
- [x] Compare the partial against what was changed form 7.10 to 7.33.1 and update the overriden file as necessary: https://github.com/projectblacklight/blacklight/compare/v7.10.0...v7.33.1
- [x] Test the changes in your browser for the views functionality is still working
- [x] If you had to update the file and there is not a pre-existing OVERRIDE notes, please add one with why the partial was overriden, if you can tell.
Example: `<%# OVERRIDE: Blacklight v7.33.1 to provide theming for the clients needs %>`

### CODE REVIEW

File: app/views/catalog/_constraints.html.erb [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<%# Override Blacklight v7.10 to to add startover and render constraints into the layout %>
+<%# OVERRIDE: Blacklight v7.33.1 to to add startover and render constraints into the layout %>
```
Comment (orangewolf) [inline app/views/catalog/_constraints.html.erb:1]: ❤️

File: app/views/catalog/show.html.erb [hunk 2]
```diff
@@ -7,7 +7,7 @@
       <%= render 'start_over' %>
       <%= link_back_to_catalog class: 'btn' if @search_context %>
     </div>
-  <% end %>
+<% end %>
 
-  <%= render_document_main_content_partial %>
+<%= render 'show_main_content' %>
```
Comment (orangewolf) [inline app/views/catalog/show.html.erb:12]: wow - I love that they are moving away from these silly view helpers!

**Review by orangewolf — APPROVED** [review]
this is very well done.

**Comment by aprilrieger** [comment]
> this is very well done.

Thank you :)

**Pull request merged**